Makefile

```diff
@@ -0,0 +1,8 @@
+update-asset-lake-schema:
+	cp ../cheese-pizza/python/asset-lake/asset-lake-api-openapi-3.1.0.json fern/apis/asset-lake/openapi/openapi.json
```

Comment:
Out of curiosity what's the difference between this file and this file `asset-lake-api-openapi.json` in the same directory?

Reply:
Different version of the openapi spec. Version `3.0.3` is the default since it was what it started out generating I think the cli or finetuning needed something from openapi 3.1 hence extra json file. Perhaps once everything is using fern we can get rid of this.